src/Partial/Unsafe.purs

```diff
@@ -9,7 +9,10 @@ module Partial.Unsafe
 import Partial (crashWith)
 
 -- | Discharge a partiality constraint, unsafely.
-foreign import unsafePartial :: forall a. (Partial => a) -> a
+foreign import _unsafePartial :: forall a b. a -> b
+
+unsafePartial :: forall a. (Partial => a) -> a
+unsafePartial = _unsafePartial
```

Comment:
I'm not sure whether `_unsafePartial`'s type signature is correct.

Reply:
It's not exactly right, since it calls `f()`, but if it was given a more accurate signature it would need to be `unsafeCoerce`d anyway, since there's no way to reify a dictionary with a value in the PS language itself.

We could perhaps note in the doc-comment for the FFI value (the current doc-comment needs moving down here by the way) that it may look like `unsafeCoerce` from the signature, but the FFI implementation is different.